src/View.Uno.Shared/Controls/ImageSlideshow/ImageSlideshow.cs

```diff
@@ -262,7 +262,7 @@ private void MoveToNext()
 			CoreDispatcher dispatcher = Dispatcher;
 			CoreDispatcherPriority priority = CoreDispatcherPriority.Normal;
 #endif
-			_ = dispatcher.RunTaskAsync(priority, async () =>
+			_ = dispatcher.RunAsync(priority, async () =>
```

Comment:
Unless you have another overload, this looks like an async void.

Reply:
Updated with new function that looks good.